extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/linkHandler/PeertubeSearchQueryHandlerFactory.java

```diff
@@ -4,25 +4,29 @@
 import org.schabi.newpipe.extractor.exceptions.ParsingException;
 import org.schabi.newpipe.extractor.linkhandler.SearchQueryHandlerFactory;
 import org.schabi.newpipe.extractor.utils.Utils;
+import org.schabi.newpipe.extractor.search.filter.FilterItem;
+import org.schabi.newpipe.extractor.services.peertube.PeertubeHelpers;
+import org.schabi.newpipe.extractor.services.peertube.search.filter.PeertubeFilters;
 
 import java.io.UnsupportedEncodingException;
 import java.util.List;
+import java.util.Optional;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
 public final class PeertubeSearchQueryHandlerFactory extends SearchQueryHandlerFactory {
 
     private static final PeertubeSearchQueryHandlerFactory INSTANCE =
             new PeertubeSearchQueryHandlerFactory();
 
     public static final String VIDEOS = "videos";
-    public static final String SEPIA_VIDEOS = "sepia_videos"; // sepia is the global index
-    public static final String PLAYLISTS = "playlists";
-    public static final String CHANNELS = "channels";
+    // sepia is the global index
     public static final String SEPIA_BASE_URL = "https://sepiasearch.org";
-    public static final String SEARCH_ENDPOINT_PLAYLISTS = "/api/v1/search/video-playlists";
-    public static final String SEARCH_ENDPOINT_VIDEOS = "/api/v1/search/videos";
-    public static final String SEARCH_ENDPOINT_CHANNELS = "/api/v1/search/video-channels";
```

Comment:
I think this broke PeerTube filters. In theory using `resultType=videos/playlists/channels` should work, however it doesn't seem to, see [query with `resultType`](https://diode.zone/api/v1/search/videos?search=linux&resultType=playlist) and [query with different endpoints](https://diode.zone/api/v1/search/video-playlists?search=linux). PeerTube on the web uses the endpoints, too.

Reply:
Fixed with 70a2b639005feea223195f7547df22d44671f966